WPMedia/fusion-news-theme-blocks
# TMEDIA-250 - Numbered List and Simple List Stories

## Description
Add stories for Numbered List and Simple List
For ease of storybook integration switching components to be functional components allows for easier storybook mocking

Card List is part of TMEDIA-250, and will coming another PR

## Jira Ticket
- [TMEDIA-250](https://arcpublishing.atlassian.net/browse/TMEDIA-250)

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-250-card-numbered-simple-list-stories`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/simple-list-block,@wpmedia/numbered-list-block`
3. Validate Numbered List and Simple list still work as expected - http://localhost/homepage/?_website=the-gazette
4. Click through to Chromatic from the pull request check lists to see stories


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
